mteb/models/sentence_transformers_models.py

```diff
@@ -165,3 +165,81 @@
     supersedes="sentence-transformers/all-mpnet-base-v1",
     adapted_from=None,
 )
+
+e5_base_v2 = ModelMeta(
+    name="intfloat/e5-base-v2",
```

Comment:
I think prompts should be added to the e5 models and moved to [e5_models.py](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/models/e5_models.py). Additionally, we could consider integrating a mechanism to fetch metadata from Hugging Face, similar to the script in the results repository for fetching external results.

Reply:
(@x-tabdeveloping did some work on the last part - we could use a process similar to the descriptive stats to avoid always downloading these). Let us make an issue on that part

Reply:
Added to e5_models.py